app/main.py

```diff
@@ -1,2 +1,133 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from fractions import Fraction
+from typing import Any, Hashable
+
+
+CAPACITY = 8
```

Comment:
These constants as well as Node class a part of dict internal implementation so it would make sense to package everything inside of Dictionary class